docs/user_guide/method/method.rst

```diff
@@ -0,0 +1,2384 @@
+into three types: (1) :py:meth:`~mrsimulator.method.SpectralEvent`, (2)
+:py:meth:`~mrsimulator.method.DelayEvent`, and (3)
+:py:meth:`~mrsimulator.method.MixingEvent`.  This ordered list of Event objects
+is used to select the desired transition pathways and determine their average
+frequency and complex amplitude in the **SpectralDimension**.
+
+.. warning::
+
+  DelayEvent objects are not available in version 0.7 of **mrsimulator**.
+
+**SpectralEvent** and **DelayEvent** objects define which transitions are
+observed during the event and under which transition-dependent frequency
+contributions they evolve. No coherence transfer among transitions or
+populations occurs in a spectral or delay event. The transition-dependent
+frequency contributions during an Event are selected from a list of
+:ref:`enumeration literals<freq_contrib_api>` and placed in the ``freq_contrib``
+attribute of the event. If ``freq_contrib`` is left unspecified, i.e., the
+value of ``freq_contrib`` is set to ``Null``, a default list holding the
+enumeration literals for *all* contributions is generated for the event.
+
+
+.. note::
+
+  All frequency contributions from direct and indirect spin-spin couplings are
+  calculated in the weak-coupling limit in **mrsimulator**.
+
+Additionally, the user can affect transition frequencies during a spectral or
+delay event by changing other measurement attributes: ``rotor_frequency``,
+``rotor_angle``, and ``magnetic_flux_density``. If left unspecified, these
+attributes default to the values of the identically named global attributes in
+the **Method** object. SpectralEvents objects use the ``fraction`` attribute to
+calculate the weighted average frequency during the spectral dimension for each
+selected transition pathway.
+
+Inside **SpectralEvent** and **DelayEvent** objects, is a list of
+:py:meth:`~mrsimulator.method.query.TransitionQuery` objects (*vide infra*)
+which determine which transitions are observed during the event. **Method**
+objects in **mrsimulator** are general-purpose because they are designed for an
+arbitrary spin system, i.e., a method does not know the spin system in advance.
+When designing a **Method** object, you cannot identify and select a transition
+through its initial and final eigenstate quantum numbers. Transition selection
+is done through **TransitionQuery** and
+:py:meth:`~mrsimulator.method.query.SymmetryQuery` objects during individual
+spectral or delay events. **TransitionQuery** objects hold a list of
+**SymmetryQuery** objects which act on specific isotopes defined by the
+``channels`` attribute in **Method**. It is only during a simulation that the
+**Method** object uses its **TransitionQuery** objects to determine the selected
+transition pathways for a given **SpinSystem** object by the initial and final
+eigenstate quantum numbers of each transition.
+
+Between adjacent SpectralEvent or DelayEvent objects in an events list,
```

Comment:
In an events list, or between spectral dimensions

Reply:
yeah, that is trickier to explain.

Reply:
I'll just delete "in an events list"